auth/standard_role_manager.cc

```diff
@@ -546,18 +546,18 @@ standard_role_manager::grant(std::string_view grantee_name, std::string_view rol
         });
     };
 
-   return when_all_succeed(check_redundant(), check_cycle()).then_unpack([this, role_name, grantee_name] {
-       return this->legacy_modify_membership(grantee_name, role_name, membership_change::add);
+    return when_all_succeed(check_redundant(), check_cycle()).then_unpack([this, role_name, grantee_name, &mc] {
+        return this->modify_membership(grantee_name, role_name, membership_change::add, mc);
```

Comment:
Nit: this additional indent is not necessary

Reply:
I think it's fixing indent as we use 4 space (at least in this file)

Reply:
Ok, fine. In that case it looks like you actually missed one line after the one you fixed (`});`).

Reply:
yes, since I didn't change this line I haven't spotted, will fix